backend/postgres/postgres.go

```diff
@@ -228,15 +232,50 @@ func CustomConnectionPool(opts *pg.Options) *pg.DB {
 		Str("as", opts.ApplicationName).
 		Msg("PostgreSQL connection pool created")
 	db := pg.Connect(opts)
-	db.OnQueryProcessed(queryLogger)
+	if cfg.LogWrite || cfg.LogRead {
+		db.OnQueryProcessed(queryLogger)
+	} else {
+		log.Logger().Warn().Msg("Connection pool has logging queries disabled completely")
```

Comment:
Is this an information that we would want to log on `Info` level, assuming it happens only once per opened connection?